src/Nethermind/Nethermind.Blockchain/Blocks/IBlockStore.cs

```diff
@@ -19,9 +19,6 @@ public interface IBlockStore
     byte[]? GetRlp(long blockNumber, Hash256 blockHash);
     ReceiptRecoveryBlock? GetReceiptRecoveryBlock(long blockNumber, Hash256 blockHash);
     void Cache(Block block);
-
-    // These two are used by blocktree. Try not to use them...
-    void SetMetadata(byte[] key, byte[] value);
-    byte[]? GetMetadata(byte[] key);
     bool HasBlock(long blockNumber, Hash256 blockHash);
+    long? LowestInsertedBodyNumber { get; set; }
```

Comment:
Does it need public set? Can it just update it internally on `Insert`?

Reply:
Can't as erafiles and portal would also use insert.